src/UnicycleGenerator.cpp

```diff
@@ -34,6 +34,7 @@ class UnicycleGenerator::UnicycleGeneratorImplementation {
     double mergePointRatioBegin = 0.5;
     double mergePointRatioEnd = 0.5;
 
+    NavigationSetup navigationConfig{NavigationSetup::ManualMode};
```

Comment:
Are we fixing the Navigation mode? Are we not supposed to read this from a config file ?

Reply:
The `NavigationSetup` enum struct is set on the TrajectoryGenerator.cpp from the `walking-controllers`.
However, with this initialization we ensure retro-compatibility with the actual master branch from `walking-controller`